RDBM/String.cpp

```diff
@@ -32,6 +32,10 @@ void String::Show(std::ostream &out)
 	out << value << " ";
 }
 
+std::string& String::Get()
```

Comment:
I believe we should get copy of string because in case when somebody change string after we get it from here, it will change inside too.